app/src/main/res/values/strings.xml

```diff
@@ -15,5 +15,13 @@
     <string name="FailureDialogText500">Server Side Error: Registration unsuccessful</string>
     <string name="FailureDialogText400">Please check format of your details: Registration unsuccessful</string>
     <string name="FailureDialogText000">Unknown Error Occurred: Registration unsuccessful</string>
+    <string name="title_activity_help">HelpActivity</string>
+    <string name="HelpscrrenQ1">What does the app do for you?</string>
+    <string name="HelpScreenAns1">The Nigeria Telemedicine App helps you avail telemedicine health services just by making a few clicks. Once your are successfully registered with us please keep your mobile phone switched On so that the doctor can contact you as early as possible. Please enter correct details of yours so that we can reach you as early as possible.</string>
```

Comment:
Replace `avail` with `use`. 

Replace `your are` with `you're`

Reply:
Done